src/components/Navbar.js

```diff
@@ -29,12 +29,22 @@ const Navbar = () => {
 
   return (
     <nav className={styles.Navbar}>
-      <Link to="/">Bookstore CMS</Link>
-      <ul className={styles.UlLinks}>
-        <li><Link to="/">BOOKS</Link></li>
-        <li><Link to="/categories">CATEGORIES</Link></li>
-      </ul>
-      <button type="button" onClick={toggleUser} className={styles.Profile}>
+      <Link to="/" className={styles.Home}>Bookstore CMS</Link>
+      <div>
+        <ul className={styles.UlLinks}>
+          <li>
+            <Link to="/" className={styles.Link}>BOOKS</Link>
+          </li>
+          <li>
+            <Link to="/categories" className={styles.Link}>CATEGORIES</Link>
+          </li>
```

Comment:
- [ ] Based on the requirements for this milestone, styling should be Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Kindly make sure that the active navbar button has brighter color and non-active greyish

Your design  | Zeplin design
------------- | -------------
<img width="377" alt="Screenshot 2022-07-14 at 18 12 42" src="https://user-images.githubusercontent.com/77038610/179003005-cc1baadb-217b-40f7-8107-252f497c7d98.png"> | <img width="279" alt="Screenshot 2022-07-15 at 16 30 51" src="https://user-images.githubusercontent.com/77038610/179223399-f40cf3de-dfed-4a21-9a2c-f995bed9af51.png">